src/main/java/tric/tricproject/Service/QuestionServiceImpl.java

```diff
@@ -26,7 +26,9 @@ public class QuestionServiceImpl implements  QuestionService{
     public Question addQuestion(String questionText, String firstAnswerText, String secondAnswerText) {
 
         List<Answer> answers =new ArrayList<>();
-        Question question = questionRepository.save(new Question(questionText));
+        Question newQuestion = new Question(questionText);
+        newQuestion.setTime(30); // default time - 30 seconds
```

Comment:
need to discuss what the default time should be, might need to be more than that